conans/test/integration/command/download/download_test.py

```diff
@@ -1,212 +1,224 @@
-        self.assertIn("Use a full package reference (preferred) or the `--package`"
-                      " command argument, but not both.", client.out)
-
-    def test_download_with_package_and_recipe_args(self):
-        client = TestClient()
-        client.run("download eigen/3.3.4@conan/stable --recipe --package fake_id",
-                   assert_error=True)
-
-        self.assertIn("ERROR: recipe parameter cannot be used together with package", client.out)
-
-    def test_download_package_argument(self):
-        server = TestServer()
-        servers = {"default": server}
-
-        client = TurboTestClient(servers=servers, users={"default": [("lasote", "mypass")]})
-
-        ref = ConanFileReference.loads("pkg/0.1@lasote/stable")
-        client.create(ref)
-        client.upload_all(ref)
-        client.remove_all()
-
-        client.run("download pkg/0.1@lasote/stable -p {}".format(NO_SETTINGS_PACKAGE_ID))
-
-        rrev = client.cache.get_latest_rrev(ref)
-        pkgids = client.cache.get_package_ids(rrev)
-        prev = client.cache.get_latest_prev(pkgids[0])
-        package_folder = client.cache.pkg_layout(prev).package()
-
-        # Check not 'No remote binary packages found' warning
-        self.assertNotIn("WARN: No remote binary packages found in remote", client.out)
-        # Check at conanfile.py is downloaded
-        self.assertTrue(os.path.exists(client.cache.ref_layout(rrev).conanfile()))
-        # Check package folder created
-        self.assertTrue(os.path.exists(package_folder))
-
-    def test_download_not_found_reference(self):
-        server = TestServer()
-        servers = {"default": server}
-        client = TurboTestClient(servers=servers, users={"default": [("lasote", "mypass")]})
-        client.run("download pkg/0.1@lasote/stable", assert_error=True)
-        self.assertIn("ERROR: Recipe not found: 'pkg/0.1@lasote/stable'", client.out)
-
-    def test_no_user_channel(self):
-        # https://github.com/conan-io/conan/issues/6009
-        server = TestServer(users={"user": "password"}, write_permissions=[("*/*@*/*", "*")])
-        client = TestClient(servers={"default": server}, users={"default": [("user", "password")]})
-        client.save({"conanfile.py": GenConanfile()})
-        client.run("create . pkg/1.0@")
-        client.run("upload * --all --confirm")
-        client.run("remove * -f")
-
-        client.run("download pkg/1.0:{}".format(NO_SETTINGS_PACKAGE_ID))
-        self.assertIn("pkg/1.0: Downloading pkg/1.0:%s" % NO_SETTINGS_PACKAGE_ID, client.out)
-        self.assertIn("pkg/1.0: Package installed %s" % NO_SETTINGS_PACKAGE_ID, client.out)
-
-        # All
-        client.run("remove * -f")
-        client.run("download pkg/1.0@")
-        self.assertIn("pkg/1.0: Downloading pkg/1.0:%s" % NO_SETTINGS_PACKAGE_ID, client.out)
-        self.assertIn("pkg/1.0: Package installed %s" % NO_SETTINGS_PACKAGE_ID, client.out)
+@pytest.fixture()
+def ref():
```

Comment:
One line fixtures introduce more complexity than they provide. Just create the ``ConanFileReference`` in every test.